ospd_openvas/wrapper.py

```diff
@@ -971,11 +971,20 @@ def get_vts_in_groups(self, filters):
 
     def get_vt_param_type(self, vtid, vt_param_id):
         """ Return the type of the vt parameter from the vts dictionary. """
+
         vt_params_list = self.vts[vtid].get("vt_params")
         if vt_params_list.get(vt_param_id):
             return vt_params_list[vt_param_id]["type"]
         return False
 
+    def get_vt_param_name(self, vtid, vt_param_id):
+        """ Return the type of the vt parameter from the vts dictionary. """
+
+        vt_params_list = self.vts[vtid].get("vt_params")
+        if vt_params_list.get(vt_param_id):
+            return vt_params_list[vt_param_id]["name"]
+        return False
```

Comment:
I am not sure about the return type. Shouldn't it be None if the param isn't available?